src/index.js

```diff
@@ -1,11 +1,11 @@
-import React from "react";
-import ReactDom from "react-dom";
-import TodoContainer from "./components/TodoContainer";
-import "./App.css"
+import React from 'react';
+import ReactDom from 'react-dom';
+import TodoContainer from './components/TodoContainer';
+import './App.css';
 
 ReactDom.render(
```

Comment:
- [Optional] In React 18 it is recommended to use `ReactDOM.createRoot` to render your root component.